pkg/webui/console/views/gateway-general-settings/basic-settings-form/validation-schema.js

```diff
@@ -84,6 +86,20 @@ const validationSchema = Yup.object().shape({
       sharedMessages.attributeValueValidateTooLong,
       attributeValueTooLongCheck,
     ),
+  administrative_contact: Yup.object()
+    .when(['organization_ids'], {
+      is: organizationIds => Boolean(organizationIds),
+      then: schema => schema.concat(organizationSchema),
+      otherwise: schema => schema.concat(userSchema),
+    })
+    .required(sharedMessages.validateRequired),
+  technical_contact: Yup.object()
+    .when(['organization_ids'], {
+      is: organizationIds => Boolean(organizationIds),
+      then: schema => schema.concat(organizationSchema),
+      otherwise: schema => schema.concat(userSchema),
+    })
+    .required(sharedMessages.validateRequired),
```

Comment:
There's a lot duplication now with this schema. I'd suggest to outsource this into a lib: `/webui/console/lib/shared-schemas.js`